pytorch_toolkit/object_detection/tests/train_tests_custom_detection.py

```diff
@@ -12,23 +12,63 @@
 # See the License for the specific language governing permissions
```

Comment:
Change the name of the file to reflect the e2e nature of tests?

Reply:
Understand your concern, but I would not change a name because it is used in tests discoverer. And this system is in redesign stage now. I would not like to change current behavior. May I leave it as is?

Reply:
Ok